api/auth/azure/go.mod

```diff
@@ -2,7 +2,4 @@ module github.com/hashicorp/vault/api/auth/azure
 
 go 1.16
 
-require (
-	github.com/Azure/go-autorest/autorest v0.11.21
```

Comment:
This got removed by `go mod tidy`, not sure if it's expected

Reply:
Looks like that was hanging on from before when the AzureAuth's `WithResource` method took in parameters from go-autorest, but it no longer does so we're good.